<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchReceiverDialog</class>
 <widget class="QDialog" name="BatchReceiverDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>批量添加接收器</string>
  </property>
  <widget class="QPushButton" name="ptnDelete">
   <property name="geometry">
    <rect>
     <x>583</x>
     <y>91</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptnModify">
   <property name="geometry">
    <rect>
     <x>502</x>
     <y>91</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptnAdd">
   <property name="geometry">
    <rect>
     <x>421</x>
     <y>91</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>增加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptnSave">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>640</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptnClose">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>640</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="receiverTableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>620</width>
     <height>470</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>接收器名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>接收器编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>信道编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>备注</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>备 注：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtReceiverComments"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>151</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>信道编号*：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtChannelID"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">color:red</string>
      </property>
      <property name="text">
       <string>由0-9的数字组成</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>161</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>接收器编号*：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtReceiverNo"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">color:red
</string>
      </property>
      <property name="text">
       <string>由0-9的数字组成,固定长3位</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>接收器名称*：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtReceiverName"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
